### PR TITLE
add minutes from 2015-11-19 meeting

### PR DESCRIPTION
**Pull request by pmuellr**

### CODE REVIEW

**Comment by trevnorris** [comment]
LGTM

File: wg-meetings/2015-11-19.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+
+FFI
+--------------------------------------------------------------------------------
+
+[YouTube 38:44](https://youtu.be/3rszjGYpbyM?t=2324)
+
+Alexis - FFI is invoking C functions from JS, most native modules could use
+this to do their work.  Some issues have come up regarding performance issues,
+but there are some approaches which minimize performance, such as
```
Comment (orangemocha) [inline wg-meetings/2015-11-19.md]: `minimize performance` => `minimize impact on performance`

File: wg-meetings/2015-11-19.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+Chakra using templates and having a compiler provide some glue.
```
Comment (orangemocha) [inline wg-meetings/2015-11-19.md]: This is not actually used by Chakra, though it was suggested by a member of the Chakra team. We can just say `as suggested at https://github.com/nodejs/nan/issues/349#issuecomment-110569177`.

**Comment by orangemocha** [comment]
Minor nits. Other than that LGTM

**Comment by mhdawson** [comment]
lgtm

**Comment by pmuellr** [comment]
Thanks everyone; merged in the suggested fixed and pushed on master.

**Pull request merged**